.github/workflows/docker_test_2.yml

```diff
@@ -1,10 +1,11 @@
-name: cluster_vtctl_web
+name: docker test 2
```

Comment:
```suggestion
name: docker_test_2
```

Reply:
this is already fixed, marking as resolved.